src/variables.jl

```diff
@@ -27,12 +34,18 @@ function JuMP.build_variable(
         _error("Invalid fix value, must be false or true.")
     elseif info.has_start && !isone(info.start) && !iszero(info.start)
         _error("Invalid start value, must be false or true.")
+    elseif (info.has_start || info.has_fix) && !isnothing(logical_compliment)
+        _error("Cannot fix or provide a start value for logical variable " *
```

Comment:
"for a logical variable that is a logical compliment variable" (added "a" and "is")